tests/e2e/e2e_test.go

```diff
@@ -116,6 +122,237 @@ var _ = ginkgo.AfterSuite(func() {
 })
 
 var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
+	ginkgo.It("can create blockhains", func() {
+		existingSubnetID := ""
+		ginkgo.By("start with blockchain specs", func() {
+			color.Outf("{{green}}sending 'start' with the valid binary path:{{/}} %q\n", execPath1)
+			ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+			resp, err := cli.Start(ctx, execPath2,
+				client.WithBlockchainSpecs([]*rpcpb.BlockchainSpec{
+					{
+						VmName:  "subnetevm",
+						Genesis: "tests/e2e/subnet-evm-genesis.json",
+					},
+				}),
+			)
+			cancel()
+			gomega.Ω(err).Should(gomega.BeNil())
+			color.Outf("{{green}}successfully started:{{/}} %+v\n", resp.ClusterInfo.NodeNames)
+		})
+
+		ginkgo.By("wait for network to be healthy", func() {
+			color.Outf("{{blue}}wait for health{{/}\n")
+			ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+			_, err := cli.Health(ctx)
+			cancel()
+			gomega.Ω(err).Should(gomega.BeNil())
+		})
+
+		ginkgo.By("get status", func() {
+			color.Outf("{{blue}}get status for existing subnet ID{{/}}\n")
+			ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+			subnetDeployed := false
+		LOOP:
+			for !subnetDeployed {
+				select {
+				case <-ctx.Done():
+					break LOOP
+				case <-time.After(5 * time.Second):
+					stat, err := cli.Status(ctx)
+					gomega.Ω(err).Should(gomega.BeNil())
+					if stat.ClusterInfo.CustomChainsHealthy {
+						subnetDeployed = true
+						existingSubnetID = stat.ClusterInfo.GetSubnets()[0]
+						gomega.Ω(existingSubnetID).Should(gomega.Not(gomega.BeNil()))
+					}
+				}
+			}
+			cancel()
+			gomega.Ω(subnetDeployed).Should(gomega.BeTrue())
+		})
+
+		time.Sleep(1 * time.Second)
+		ginkgo.By("can create a blockchain with a new subnet id", func() {
+			color.Outf("{{blue}}can create a blockchain in a new subnet{{/}}\n")
+			ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+			_, err := cli.CreateBlockchains(ctx,
+				[]*rpcpb.BlockchainSpec{
+					{
+						VmName:  "subnetevm",
+						Genesis: "tests/e2e/subnet-evm-genesis.json",
+					},
+				},
+			)
+			cancel()
+			gomega.Ω(err).Should(gomega.BeNil())
+		})
+
+		ginkgo.By("wait for network to be healthy", func() {
+			color.Outf("{{blue}}wait for health{{/}\n")
```

Comment:
you should use CustomChainsHealthy wait here, this is error prone for the moment